Rocket/Rocket.API/Collections/TranslationList.cs

```diff
@@ -66,16 +87,6 @@ public void Add(string key, string value)
             translations.Add(new TranslationListEntry(key, value));
         }
 
-        public void AddRange(IEnumerable<TranslationListEntry> collection)
-        {
-            translations.AddRange(collection);
-        }
-
-        public void AddRange(TranslationList collection)
-        {
-            translations.AddRange(collection.translations);
-        }
-
```

Comment:
I don't think so, have you ever seen a plugin that uses that method to add translations?

Reply:
Maybe you're right that you and I didn't saw that someone used/use it, but I still think it makes sense to keep it, just in case, who knows, maybe someone used it or still use

Reply:
Added those methods back in the latest commit